layouts/partials/head.html

```diff
@@ -0,0 +1,57 @@
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+{{ hugo.Generator }}
+{{- $outputFormat := partial "outputformat.html" . -}}
+
+{{ range .AlternativeOutputFormats -}}
+<link rel="{{ .Rel }}" type="{{ .MediaType.Type }}" href="{{ .Permalink | safeURL }}">
+{{ end -}}
+
+{{ if and (eq (getenv "HUGO_ENV") "production") (ne $outputFormat "print") -}}
+<META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
+{{ else -}}
+<META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
+{{ end -}}
+
+{{ partialCached "favicons.html" . }}
+<title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{ . }} | {{ end }}{{ .Site.Title }}{{ end }}</title>
+<meta name="description" content="{{ .Site.Params.Description }}">
+
+
+<meta property="og:title" content="Flux - the GitOps family of projects" />
+<meta property="og:description" content="<h1>Flux</h1> is a <strong>set</set> of <pre>continuous</pre> and progressive delivery solutions for Kubernetes, <h3>and</h3> they are open and extensible. 
+The APIs of Flux are stable now. Now is the best time to start using Flux, or start your migration if you are a legacy user." />
+<meta property="og:type" content="website" />
+<meta property="og:url" content="//localhost:1313/" /><meta property="og:image" content="//localhost:1313/img/flux-social.png"/>
```

Comment:
Maybe link to `/` as opposed to localhost?

Reply:
I pushed a change just to try it out, but I feel that slack and twitter probably will not render any markup in the op:description meta tag. I did not find references describing that. 

https://ogp.me/
https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup

Reply:
With url fixes:
![Screenshot from 2021-08-13 13-14-10](https://user-images.githubusercontent.com/4100/129342403-4482cb72-e401-46a0-94a4-e1dc49200592.png)
![image](https://user-images.githubusercontent.com/4100/129342484-92f16a06-a99a-4d0b-8247-922508214d61.png)